components/bio-formats-tools/src/loci/formats/tools/ImageConverter.java

```diff
@@ -262,6 +298,16 @@ private void printUsage() {
       "     -padded: filename indexes for series, z, c and t will be zero padded",
       "     -option: add the specified key/value pair to the options list",
       "",
+      "The extension of the output file specifies the file format to use",
+      "for the conversion. The list of available formats and extensions is:",
+      "",
+      listExtensions(),
+      "",
```

Comment:
Don't need these blanks after the `list*` because already `println`'ing strings that end with `\n`.